proxy/send.go

```diff
@@ -0,0 +1,179 @@
+// send.go - mix network client send
+// Copyright (C) 2017  David Anthony Stainton
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package proxy
+
+import (
+	"sync"
+	"time"
+
+	"github.com/katzenpost/client/crypto/block"
+	"github.com/katzenpost/client/path_selection"
+	"github.com/katzenpost/client/scheduler"
+	"github.com/katzenpost/client/session_pool"
+	"github.com/katzenpost/client/storage"
+	"github.com/katzenpost/client/user_pki"
+	"github.com/katzenpost/core/crypto/rand"
+	"github.com/katzenpost/core/sphinx"
+	"github.com/katzenpost/core/sphinx/constants"
+	"github.com/katzenpost/core/wire"
+	"github.com/katzenpost/core/wire/commands"
+)
+
+const (
+	RoundTripTimeSlop = 3 * time.Minute // XXX fix me
```

Comment:
should this go in consts, rather than this file?

Reply:
yes it should. good catch.